commands/reports/switch_binary.go

```diff
@@ -0,0 +1,25 @@
+package reports
+
+import "fmt"
+
+type SwitchBinary struct {
+	*report
+	Status bool `json:"status"`
+	data   []byte
+}
+
+// NewSwitchBinary creates a new switchBinary report that contains the switch status from a zwave node
+func NewSwitchBinary(data []byte) (*SwitchBinary, error) {
+	if len(data) < 1 {
+		data = []byte{0x00}
```

Comment:
When I read the manual I discovered that we have totally missed version 2 of the switch_binary report. I added it, updated the tests and added error returns when length is invalid.